deploy/002_dca_hub_position_descriptor_fixed.ts

```diff
@@ -0,0 +1,75 @@
+import { DeployFunction } from '@0xged/hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { bytecode } from '../artifacts/solidity/contracts/DCAHubPositionDescriptorFixed.sol/DCAHubPositionDescriptorFixed.json';
+import { deployThroughDeterministicFactory } from '@mean-finance/deterministic-factory/utils/deployment';
+
+const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployer } = await hre.getNamedAccounts();
+
+  let msig: string | undefined;
+  switch (hre.deployments.getNetworkName().toLowerCase()) {
```

Comment:
Instead of having the `switch` here, let's use named accounts, like the deployer. 
Like [here](https://github.com/Balmy-protocol/dca-v2-core/blob/main/hardhat.config.ts#L99) and [here](https://github.com/Balmy-protocol/dca-v2-core/blob/main/deploy/001_permission_manager.ts#L7)